test/acceptance/behave/README.md

```diff
@@ -156,7 +164,8 @@ Feature: feature name...
   - If attribute number is "1", the attribute name is without consecutive, ex: `attributes_name=temperature`
     Else attributes number is major than "1" the attributes name are prefix plus consecutive, ex:
         `attributes_name=temperature_0, attributes_name=temperature_1, ..., temperature_N`
-  - If would like a wrong query parameter name, use `qp_` prefix       
+  - If would like a wrong query parameter name, use `qp_` prefix   
+  - the `-harakiri` option is used to kill contextBroker (must be compiled in DEBUG mode)
```

Comment:
Why stopping CB using regular kill command (as the testHarness.sh suite does) is not suitable in this case?